src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
Overall this is code duplication since AFAIK the storage proposal has boot requirement checkers.
Those should be used.